### ISSUE TITLE
Option to expose MANRS, reseller and resold tags in JSON Export

### ISSUE THREAD

**Issue by listerr** [inex/IXP-Manager#674]
**Is your feature request related to a problem? Please describe.**

We would like to update our website to show the MANRS label for members who have joined the MANRS programme, and
also it would be useful in other places to be able to see if a member is a reseller or resold in the JSON export.

**Describe the solution you'd like**

There is an existing switch to show customer tags:

https://ixp.example.com/api/v4/member-export/ixf/0.7?withtags=1

Perhaps this could be implemented as an additional tag, or a similar option e.g. ?withmanrs=1

**Describe alternatives you've considered**

- Is there a way to do this using the member template/skin documented at 
https://docs.ixpmanager.org/features/member-export/ ?

- A script to identify all MANRS / resold / reseller members and add a member tag to them, therefore they can be found
with the ?withtags=1, but this obviously means duplicating data.

Thanks,

Rob

**Comment by listerr** [comment]
Actually, this turned out to be fairly easy to do with the skin, as the functions are already there in /database/Entities/Customer.php.

Based on the example : `resources/skins/example/content/members/3/json-example.foil.php`
I copied this to our skin and added the following lines:

```
    $data[ $c->getId() ]['in_manrs']         = $c->getInManrs();
    $data[ $c->getId() ]['is_resold']        = $c->isResoldCustomer();
    $data[ $c->getId() ]['is_reseller']      = $c->getIsReseller();
```

So this just spits out JSON with those fields included for each member.

**Comment by barryo** [comment]
This is now implemented and [documented here](http://127.0.0.1:8000/features/ixf-export/#including-ixp-manager-specific-data).

I've kept this all under the `withtags=1` mechanism and within the `ixpmanager` section as it is not part of the IX-F Export Schema.
